### ISSUE TITLE
Recheck `Switch` accessibility

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#459]
`isSelected` trait seems to be missing. Other accessibility properties should be rechecked as well.
